packages/squiggle-lang/src/rescript/Distributions/GenericDist.res

```diff
@@ -159,6 +119,89 @@ let toPointSet = (
   }
 }
 
+module Score = {
+  type scoreDistOrScalar = DistributionTypes.DistributionOperation.scoreDistOrScalar
+
+  type pointSet_ScoreDistOrScalar = D(PointSetTypes.pointSetDist) | S(float)
```

Comment:
Instead of underscore, use modules.
``pointSet_ScoreDistOrScalar`` -> ``module PointSet = {}``